### DISCUSSION TITLE
GCP sign in glitch.

### DISCUSSION THREAD

**Discussion by JettSen01** [AntlerVC/firetable#1313] [Support / Q&A] [answered]
So, I created a Rowy account and added the PRO plan. 

I built the first Workspace project and connected to Google and Firebase perfectly fine. I wanted to start fresh so I deleted the project from both Firebase and Rowy and now I cant create or start any projects at all.

The issue I get is:
- Create Project fine
- Sign in with Google fine
- It even shows I have signed in
- Click 'Proceed' - the screen takes me to the first page to select database **for a split second flash** and then back to 'Sign in with Google'.

So, my account is completely useless now because I cant do anything till I do this step.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @JettSen01, could you follow the steps mentioned in this doc: https://docs.rowy.io/faqs/deleted-project. For the 4th step, we'd need the Project Owner's confirmation over email to support@rowy.io. Once this is done, we can process the project deletion for you on our side.

**Reply by JettSen01** [reply]
@BeeBombshell … thank you I will give this a try and report back 👍🏻

**Reply by JettSen01** [reply]
@BeeBombshell  I did all the steps to no avail.

**Reply by BeeBombshell** [reply]
@JettSen01, not sure if I'm able to find your email on support@rowy.io. Could you share your email id so that I can check?

**Comment by JettSen01** [comment]
Hi Bhavya,My email is as above. ***@***.*** 10 Jul 2023, at 8:43 pm, Bhavya Verma ***@***.***> wrote:﻿
@JettSen01, not sure if I'm able to find your email on ***@***.*** Could you share your email id so that I can check?
